spec/models/comment_spec.rb

```diff
@@ -0,0 +1 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly add unit test for comment model

Reply:
Thank you ,its done